src/utils.rs

```diff
@@ -52,26 +52,27 @@ pub fn virtual_keycode_is_char(vk: VirtualKeyCode) -> bool {
 		VirtualKeyCode::Numpad9 => true,
 		VirtualKeyCode::AbntC1 => true,
 		VirtualKeyCode::AbntC2 => true,
-		VirtualKeyCode::Add => true,
 		VirtualKeyCode::Apostrophe => true,
 		VirtualKeyCode::At => true,
```

Comment:
Add the missing `Asterisk` variant here (which is '*'). This actually shouldn't make a difference in how emulsion works, but let's keep things consistent.

Reply:
Done